src/main/scala/org/globalforestwatch/features/FeatureRDD.scala

```diff
@@ -22,14 +30,18 @@ object FeatureRDD {
     val featuresDF: DataFrame =
       FeatureDF(input, featureObj, kwargs, spark)
 
-    featuresDF.rdd.mapPartitions({ iter: Iterator[Row] =>
-      for {
-        i <- iter
-        if featureObj.isValidGeom(i)
-      } yield {
-        featureObj.get(i)
+    featuresDF.rdd
+      .mapPartitions({ iter: Iterator[Row] =>
+        for {
+          i <- iter
+          if featureObj.isValidGeom(i)
+        } yield {
+          featureObj.get(i)
+        }
+      }, preservesPartitioning = true)
+      .flatMap { feature =>
```

Comment:
Do we plan to export a reusable TSV at the end for recurring updates of the same lists, or we will always just generate them from scratch?